charts/rollout-operator/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 name: rollout-operator
 description: "Grafana rollout-operator"
 type: application
-version: 0.1.0
+version: 0.1.1
```

Comment:
Is it possible that this PR breaks existing Helm installs, or makes the upgrades much more painful? What would happen if someone was already using with chart name `rollout_operator`? I am not so certain how likely that scenario is either.

Reply:
It's not possible to use with rollout_operator name because you get:
```
Error: INSTALLATION FAILED: ServiceAccount "krajo-rollout_operator" is invalid: metadata.name: Invalid value: "krajo-rollout_operator": a lowercase RFC 1123 subdomain must consist of lower case alphanumeric characters, '-' or '.', and must start and end with an alphanumeric character (e.g. 'example.com', regex used for validation is '[a-z0-9]([-a-z0-9]*[a-z0-9])?(\.[a-z0-9]([-a-z0-9]*[a-z0-9])?)*')
```
This came about because of subcharting, see Chart.yaml here: https://github.com/grafana/helm-charts/pull/1205/files

```
  - name: rollout-operator
    alias: rollout_operator
    version: 0.1.1
    repository: https://grafana.github.io/helm-charts
    condition: rollout_operator.enabled
```

Reply:
The alternative is to use nameOverride instead in the chart that is using this , hmm?

Reply:
ok I see now. This PR makes using `rollout_operator` as chart name possible. I am ok with the current approach.